.github/workflows/trymerge.yml

```diff
@@ -29,9 +29,12 @@ jobs:
           PR_NUM: ${{ github.event.client_payload.pr_num }}
           GH_RUN_URL: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
           FORCE: ${{ github.event.client_payload.force}}
+          ON_GREEN: ${{ github.event.client_payload.on_green}}
         run: |
           if [ -n "${FORCE}" ]; then
             python3 .github/scripts/trymerge.py --force "${PR_NUM}"
+          elif [-n "${ON_GREEN}"]; then
+            python3. github/scripts/trymerge.py --onGreen "${PR_NUM}"
```

Comment:
I believe you would need a corresponding flag in parse_args(), and the convention is usually --on-green (vs the camel case)